### PR TITLE
chore: install clasee cla check github action

### PR DESCRIPTION
**Pull request by GeorgeMac**
Installs CLAsee our CLA checking github action.

- [x] Configure `CLASEE_SECRET` github secret in this repo

### Done checklist
- [x] ~docs/SPEC.md updated~
- [x] ~Test cases written~

### CODE REVIEW

**Comment by jsternberg** [comment]
Assigning myself as a responsible party to making sure this is done.

File: .github/workflows/cla.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+jobs:
+  cla-checker:
+    runs-on: ubuntu-latest
+    name: "Check CLA"
+    steps:
+      - name: "Lookup PR Author"
+        uses: influxdata/clasee@v1
+        with:
+          spreadsheet: "1jnRZYSw83oa6hcEBb1lxK6nNvXrWnOzPT8Bz9iR4Q8s"
```
Comment (jsternberg) [inline .github/workflows/cla.yml]: Is this value meant to be secret?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -2,3 +2,4 @@
 ### Done checklist
 - [ ] docs/SPEC.md updated
 - [ ] Test cases written
+- [ ] Sign [CLA](https://www.influxdata.com/legal/cla/) (if not already signed)
```
Comment (jsternberg) [inline .github/pull_request_template.md]: Do we need this anymore or can we modify the classee checker to post a comment?

Comment (GeorgeMac) [reply .github/pull_request_template.md]: I would love to get it to post a comment actually. It doesn't do that yet sadly. I will look into adding that.

**Comment by GeorgeMac** [comment]
holding off on this as I overlooked access to secrets by forks (there is no access basically)

**Pull request closed without merging**